src/mp-instance.js

```diff
@@ -1302,12 +1304,21 @@ function completeSDKInitialization(apiKey, config, mpInstance) {
         }
     }
 
+    // add a new function to apply items to the store that require config to be returned
     mpInstance._Store.storageName = mpInstance._Helpers.createMainStorageName(
         config.workspaceToken
     );
     mpInstance._Store.prodStorageName = mpInstance._Helpers.createProductStorageName(
         config.workspaceToken
     );
+    mpInstance._Identity.idCache = new LocalStorageVault(
```

Comment:
add comment to explain why this is happening here and not in identity.js.
```suggestion
// idCache deepnds on store.storageName, which doesn't exist when identity.js gets instantiated.  identity.js gets instantiated right when mp is initialized, but store doesnt' get instantiated until later.
    mpInstance._Identity.idCache = new LocalStorageVault(
```